freeipa/src/main/resources/application.yml

```diff
@@ -207,3 +207,10 @@ cb:
       enabled: true
       contentlogging: true
 
+clusterProxy:
+  url: http://localhost:10080/cluster-proxy
+  enabled: false
```

Comment:
@keyki How do we go about enabling cluster proxy integration in mow-dev, int, etc?

Reply:
@aarman-cloudera These configs are used by the FMS in mow. Can you flip enabled to true and confirm whether localhost is the right host to talk to the cluster proxy?

Reply:
As per discussion with David, we'll look into enabling it in mow-dev, int, etc in a follow on commit.